web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -37,58 +37,44 @@
 import com.fasterxml.jackson.databind.SerializationFeature;
 import java.util.HashMap;
 import java.util.Map;
-import org.cbioportal.model.*;
```

Comment:
I'm surprised I didn't notice this duplication before .. this is a good reason to sort import statements alphabetically.